src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -741,6 +741,24 @@ public void WriteDatabaseRecord(DatabaseRecord databaseRecord, SmugglerProgressB
                     progress.SqlEtlsUpdated = true;
                 }
 
+                if (databaseRecord?.TimeSeries != null)
+                {
+                    if (currentDatabaseRecord?.TimeSeries != null)
+                    {
+                        foreach (var collection in currentDatabaseRecord.TimeSeries.Collections)
+                        {
+                            if ((databaseRecord.TimeSeries.Collections.ContainsKey(collection.Key)) == false)
```

Comment:
Why this check?

Reply:
This is that check we do for revision configuration as well. 
I guess we don't want to override an existing configuration? But I agree that it might result in an unexpected behavior without any warning/message. 
What do we want to do here? overwrite or notify that we skip this config, because one is already exists?

Reply:
I think this was originally meant for import, so if you update an existing value?